packages/iov-ethereum/src/abi.spec.ts

```diff
@@ -6,6 +6,15 @@ import { Abi } from "./abi";
 const { fromHex } = Encoding;
 
 describe("Abi", () => {
+  describe("calculateMethodId", () => {
+    it("works", () => {
+      // Examples from https://solidity.readthedocs.io/en/develop/abi-spec.html#examples
```

Comment:
Is it possible to fix this to a version? E.g. v0.5.7

Reply:
Good idea, done